File: 1791-richest-customer-wealth/1791-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max = Integer.MIN_VALUE;
        int val = 0;
        for(int[] row : accounts){
            for(int cols : row){
                val += cols;
            }
            if(val > max) max = val;
            val = 0;
        }
        return max;
    }
}

File: 1791-richest-customer-wealth/1791-richest-customer-wealth.c
int maximumWealth(int** arr, int m, int* n) {
    int  max = 0, sum;
    // printf("%d %d", m , *n);
    for(int i = 0; i < m; i++){
        sum = 0;
        for (int j = 0; j < *n; j++){
            sum += arr[i][j];
        }
        if(sum > max) max = sum;
    }
    return max;
}